core/Auth/Password.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
FYI: We need to set Version to 3.0.4-b4 in core/Version.php . We usually don't do this in PRs already but I think in this case it is a good idea so the update gets executed in the tests etc.